### ISSUE TITLE
[a11y] 1.1.1 Non-text content

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/client#11527]
### Describe the bug

![screenshot_002399](https://github.com/owncloud/client/assets/26610733/a2ede82f-fac2-4ce2-aa33-7a1a23f81387)

- [x] #11665

![screenshot_002400](https://github.com/owncloud/client/assets/26610733/ca728477-8d07-43d9-b237-7de9657b9a57)
- [x] The arrows "<" and ">" should be hidden by screen readers. This ensures ensures that NVDA or other screen readers do not display the arrows as "Size/Smaller than" or similar text output.

![screenshot_002401](https://github.com/owncloud/client/assets/26610733/50388c4e-f21a-4d2a-82c5-81d2bd63d8c6)

- [x] Images that convey the visual status of the space are not recognized by screen readers and are not displayed. If the Space is not available, an error message an error message is normally displayed. However, this error message was not displayed in the test either

![screenshot_002402](https://github.com/owncloud/client/assets/26610733/47cb42c2-65d4-495a-89e4-bdb797e9ebd9)

- [x] #11678

![screenshot_002403](https://github.com/owncloud/client/assets/26610733/959ee0ec-e110-472b-bece-1027c5bf78d9)

- [x] The Space images in the table column "Image" are not provided with alternative texts.

### Expected behavior

Comply with https://www.bit-inklusiv.de/bitv-softwaretest/1-1-1-nicht-text-inhalte/

### Client version number

4.0.0.10514-alpha 1
